tests/integration/apply_test.go

```diff
@@ -64,4 +78,28 @@ func Test_Apply_3x(t *testing.T) {
 			assert.Equal(t, expected, out)
 		})
 	}
+
+	t.Run("updates existing entities", func(t *testing.T) {
+		runWhen(t, "kong", ">=3.0.0")
+		setup(t)
+
+		apply("testdata/apply/007-update-existing-entity/service-01.yaml")
+
+		out, _ := dump()
```

Comment:
Why not check the error?

```suggestion
		out, err := dump()
		require.NoError(t, err)
```